### PR TITLE
Expose SDK Metadata

### PR DESCRIPTION
**Pull request by jumaallan**
Story: N/A

## Summary

Expose the `sourceSdk` and `sourceSdkVersion` to allow them to be overridden on Flutter side

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Networking/Models/EnhancedKyc.swift [hunk 1]
```diff
@@ -30,6 +30,8 @@ public struct EnhancedKycRequest: Codable {
         bankCode: String? = nil,
         callbackUrl: String?,
         partnerParams: PartnerParams,
+        sourceSdk: String,
+        sourceSdkVersion: String,
```
Comment (JubrilO) [inline Sources/SmileID/Classes/Networking/Models/EnhancedKyc.swift:34]: I think we should set default values for these properties in the initializer parameter and remove the values where the properties are declared.

Comment (jumaallan) [reply Sources/SmileID/Classes/Networking/Models/EnhancedKyc.swift]: Fixed that

**Review by JubrilO — APPROVED** [review]
